src/components/CustomerQuota/GovWallet/IncorrectBalanceCard.tsx

```diff
@@ -51,14 +55,15 @@ interface GovWalletIncorrectBalanceCard {
   ids: string[];
   onCancel: () => void;
   govWalletBalanceInCents: number;
+  lastModifiedDate: string;
 }
 
 /**
  * This card is shown when the user balance is not accurate in GovWallet
  */
 export const GovWalletIncorrectBalanceCard: FunctionComponent<
   GovWalletIncorrectBalanceCard
-> = ({ ids, onCancel, govWalletBalanceInCents }) => {
+> = ({ ids, onCancel, govWalletBalanceInCents, lastModifiedDate }) => {
```

Comment:
ehh do we have a test to ensure the modified date time is appearing on the screen?

Reply:
noticed there isnt a IncorrectBalanceCard.test.tsx 🌚, i guess it's slightly out of the scope for this story, ok to accept for now